### PR TITLE
Fixing input and fight button in game

### PR DESCRIPTION
**Pull request by KajaKoren**
Not completed yet, quite a few things still needed to be added. Like the function on the admin button so I can tell that the admin is done with moving the robots, when that is implemented, the input field should be disabled/enabled based on if the fight button is pressed and if the admin is done with moving.

### CODE REVIEW

File: src/app/pages/Game.tsx [hunk 1]
```diff
@@ -193,6 +214,7 @@ export const Game = () => {
 					onUpdate={setRobotMovements}
 					key={JSON.stringify(robotMovements)}
 					teamColor={teamColor(selectedTeam)}
+					readyToPlay={readyToMove}
```
Comment (coderbyheart) [inline src/app/pages/Game.tsx]: I'd hide the entire form if the team is ready to play. Requires less code in the form component.

File: src/app/pages/Admin.tsx [hunk 2]
```diff
@@ -22,6 +22,7 @@ export const Admin = () => {
 
 	const [selectedRobot, setSelectedRobot] = useState<string>()
 	const [blockScroll, allowScroll] = useScrollBlock()
+	const [moveRobotsUnlocked, setMoveRobotsUnlocked] = useState<boolean>(false)
```
Comment (coderbyheart) [inline src/app/pages/Admin.tsx:37]: Should be derived from the GameEngine state.

File: src/app/pages/Game.tsx [hunk 3]
```diff
@@ -40,6 +40,10 @@ export const Game = () => {
 
 	const [blockScroll, allowScroll] = useScrollBlock()
 	const [activeRobot, setActiveRobot] = useState<string>()
+	const [fightUnlocked, setFightUnlocked] = useState<boolean>(false)
+
+	// Used to test if I could disable the inputs in robotconfig
+	const [readyToMove, setReadyToMove] = useState<boolean>(true)
```
Comment (coderbyheart) [inline src/app/pages/Game.tsx]: Should be derived from the GameEngine state.

**Review by coderbyheart — CHANGES_REQUESTED** [review]

**Review by coderbyheart — COMMENTED** [review]
- Please change your commiting email address

File: src/app/pages/Game.tsx (hunk 3 above)
Comment (KajaKoren) [reply src/app/pages/Game.tsx]: Not seeing how I can disable the fightUnlocked, as this communicates between the switch and the button

File: src/app/pages/Game.tsx (hunk 1 above)
Comment (KajaKoren) [reply src/app/pages/Game.tsx]: Should be fixed now

File: src/app/pages/Admin.tsx (hunk 2 above)
Comment (KajaKoren) [reply src/app/pages/Admin.tsx:37]: Should be fixed

File: src/app/pages/Game.tsx (hunk 3 above)
Comment (coderbyheart) [reply src/app/pages/Game.tsx]: Yes, correct. My bad.

**Pull request merged**